ww4ff/data/transform/base.py

```diff
@@ -134,7 +136,10 @@ def __call__(self, examples: Sequence[WakeWordClipExample]) -> ClassificationBat
                 new_examples.append((self.negative_label, ex.emplaced_audio_data(ex.audio_data[..., a:b])))
         new_examples = sorted(new_examples, key=lambda x: x[1].audio_data.size()[-1], reverse=True)
         lengths = torch.tensor([ex.audio_data.size(-1) for _, ex in new_examples])
-        max_length = max(ex.audio_data.size(-1) for _, ex in new_examples)
+        if self.pad_to_window:
+            max_length = int(self.sample_rate * self.window_size_ms / 1000)
+        else:
+            max_length = max(ex.audio_data.size(-1) for _, ex in new_examples)
```

Comment:
what if the audio size > max_length?

Reply:
That should never happen, since the audio is truncated to no more than window_size_ms before that line.